### ISSUE TITLE
[Manager] topic could not stop after add link from stream

### ISSUE THREAD

**Issue by saivirtue** [oharastream/ohara#4893]
![asdasdasd](https://user-images.githubusercontent.com/6234750/82117389-2fc29980-97a2-11ea-9e74-6b5797d239ca.png)

當一個topic被stream所連結的時候，點選"刪除topic"就會停在上述畫面直到噴出失敗

_Originally posted by @chia7712 in https://github.com/oharastream/ohara/issues/4539#issuecomment-629623556_

**Comment by konekoya** [comment]
@saivirtue Do you mind if i take over this issue :)

**Comment by saivirtue** [comment]
> @saivirtue Do you mind if i take over this issue :)

My pleasure

**Comment by konekoya** [comment]
@saivirtue Are you able to repro this? For me it's less often, but i can run into this once in a while 🤔

**Comment by saivirtue** [comment]
Hmm...after #4946 and #4917, this issue should not exist, right?
Could you update the latest images and give it a shoot?

**Comment by konekoya** [comment]
Okay, i will give this a shot later today :)

**Comment by konekoya** [comment]
Hmm, this still happens from time to time, but it's less often. This will require some more digging...

**Comment by saivirtue** [comment]
> Hmm, this still happens from time to time, but it's less often. This will require some more digging...

Let me dig it also :)

**Comment by konekoya** [comment]
Big thanks for this @saivirtue 🤟

**Comment by saivirtue** [comment]
/cc @chia7712 
To reproduce:
- create workspace and pipeline
- drag a stream component
- drag a pipelineOnly topic
- edit stream property, select the "to" topic to the new added topic
- close the dialog and delete the topic (must be quickly)

the configurator error:
```
2020-05-27 08:09:18,908 WARN  [kafka-admin-client-thread | adminclient-37] clients.NetworkClient$DefaultMetadataUpdater(1070): [Consumer clientId=df9da24429c0455f82259e0b6bd2c106, groupId=null] Error while fetching metadata with correlation id 16 : {3e1117d3c-c7k7o=LEADER_NOT_AVAILABLE}
```
the api response of topics:
```json
[
    {
        "displayName": "T1",
        "lastModified": 1590566945016,
        "tags": {},
        "id": "c67fcf15-44ae-419b-8d5e-573d128fdc84",
        "isShared": false,
        "group": "3e1117d3c",
        "numberOfPartitions": 1,
        "brokerClusterKey": {
            "group": "broker",
            "name": "workspace1"
        },
        "name": "c7k7o",
        "partitionInfos": [
            {
                "beginningOffset": 0,
                "endOffset": 0,
                "id": 0,
                "inSyncReplicas": [
                    {
                        "host": "dockermachine",
                        "id": 1001,
                        "port": 37083
                    }
                ],
                "leader": {
                    "host": "dockermachine",
                    "id": 1001,
                    "port": 37083
                },
                "replicas": [
                    {
                        "host": "dockermachine",
                        "id": 1001,
                        "port": 37083
                    }
                ]
            }
        ],
        "state": "RUNNING",
        "numberOfReplications": 1,
        "nodeMetrics": {
            "dockermachine": {
                "meters": []
            }
        },
        "segment.bytes": 1073741824,
        "segment.ms": 604800000
    }
]
```

**Comment by saivirtue** [comment]
Since this issue could not reproduce easily and there is a better solution base on the discussion of the meeting. I have opened a new issue to apply the conclusion. (#5083)

/cc @oharastream/developer

**Comment by konekoya** [comment]
Thanks for helping this out @saivirtue. Really a tricky one
